deploy/README.md

```diff
@@ -14,7 +14,7 @@
 * Below is the document containing steps to create and configure K8 cluster.
   * __Onprem RKE CLuster__ : Create RKE K8 cluster using mentioned [steps](https://github.com/mosip/k8s-infra/tree/v1.2.0.2/mosip/on-prem#mosip-k8s-cluster-setup-using-rke).
       * __Persistence__ : Setup storage class as per [steps](https://github.com/mosip/k8s-infra/tree/v1.2.0.1/mosip/on-prem#storage-classes).
-      * __Istio service mesh__ : Setup Istio service mesh using [steps](https://github.com/mosip/k8s-infra/tree/v1.2.0.2/mosip/on-prem#istio-for-service-discovery-and-ingress).
+      * __Istio service mesh__ : Setup Istio service mesh using [steps](https://github.com/mosip/esignet/blob/release-1.5.x/deploy/istio-gateway/install.sh).
```

Comment:
revert this as this is still open for discussion.

Reply:
reverted